js/common.js

```diff
@@ -1007,3 +1002,7 @@ var getTextWithoutDiacriticalMarks = function (text) {
    // They are removed to keep only chars without their diacritical mark.
    return text.replace(/[\u0300-\u036f]/g, '');
 }
+
+var escapeMarkupText = function (text) {
+   return jQuery.fn.select2.defaults.defaults.escapeMarkup(text);
```

Comment:
Data created in old GLPI versions may be "twice" escaped in DB, so you should keep this check to prevent reescaping an already escaped string.
```suggestion
   if (text.indexOf('>') !== -1 || text.indexOf('<') !== -1) {	
      // escape text, if it contains chevrons (can already be escaped prior to this point :/)
      text = jQuery.fn.select2.defaults.defaults.escapeMarkup(text);
   }
   return text;
```